Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -1121,21 +1121,28 @@ def add_tag(demisto_args: dict, is_attribute=False):
     uuid = demisto_args.get('uuid')
     tag = demisto_args.get('tag')
     is_local_tag = argToBoolean(demisto_args.get('is_local', False))
+    disable_output = argToBoolean(demisto_args.get('disable_output', True))
```

Comment:
the default value can be True to disable the information my default and just print the success message, as displaying displaying information would increase the runtime and may not be necessary.

But I can change it to False if you recommend to do it

Reply:
@sudkumar1 

We can't change it to default to True,
Because there are customers with playbooks that are built on this command, and this will break their playbooks,

So please change it back to the default False in the `yml` file as well.

Thanks

Reply:
@israelpoli 

understood, I have change the default value to be False.

Thanks